app/features/send/use-get-invoice-from-lud16.ts

```diff
@@ -0,0 +1,18 @@
+import { useMutation } from '@tanstack/react-query';
+import { getInvoiceFromLud16, isLNURLError } from '~/lib/lnurl';
+import type { Money } from '~/lib/money';
+
+export function useGetInvoiceFromLud16() {
+  return useMutation({
```

Comment:
this seems more like a query than mutation so should use useQuery. you were thinking mutation because it creates the ln invoice?